content/posts/numpy/numpy-rng/index.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Good practices with NumPy random number generators"
+title: "Best Practices for Using NumPy's Random Number Generators"
 date: 2024-01-26T23:22:46+02:00
 draft: false
 description: "
-Good practices with NumPy random number generators
+Best Practices for Using NumPy's Random Number Generators
 "
 tags: [tutorials, numpy, rng]
 displayInList: true
 author: ["Albert Thomas"]
 ---
 
-I want to share here what I have learned about good practices with pseudo RNGs and especially the ones available in [NumPy](https://numpy.org/).
+I want to share here what I have learned about best practices with pseudo RNGs and especially the ones available in [NumPy](https://numpy.org/).
 
-A pseudo RNG works by updating an internal state through a deterministic algorithm. The internal state is initialized with an initial value known as a seed and each update of this internal state produces a number that appears randomly generated. The key here is that the process is deterministic, meaning that if you start with the same seed and apply the same algorithm, you will get the same sequence of internal states (and numbers). Despite this determinism, the resulting numbers exhibit properties of randomness, appearing unpredictable and evenly distributed. Users can either specify the seed manually, providing a degree of control over the generated sequence, or they can opt to let the RNG object automatically derive the seed from system entropy. The latter approach enhances unpredictability by incorporating external factors into the seed.
+A pseudo RNG works by updating an internal state through a deterministic algorithm. The internal state is initialized with a value known as a seed and each update of this internal state produces a number that appears randomly generated. The key here is that the process is deterministic, meaning that if you start with the same seed and apply the same algorithm, you will get the same sequence of internal states (and numbers). Despite this determinism, the resulting numbers exhibit properties of randomness, appearing unpredictable and evenly distributed. Users can either specify the seed manually, providing a degree of control over the generated sequence, or they can opt to let the RNG object automatically derive the seed from system entropy. The latter approach enhances unpredictability by incorporating external factors into the seed.
```

Comment:
Remember to say what RNG stands for and what it is, before diving in

Reply:
indeed, done :)